scripts/Package_BGA/bga_xilinx.yml

```diff
@@ -150,3 +150,60 @@ Xilinx_FFG1156:
   mask_margin: 0.05
   layout_x: 34
   layout_y: 34
+  
+# Spartan-7 FPGA Packages
+# Notes:
+Xilinx_CPGA196:
+  description: "Spartan-7 BGA, 14x14 grid, 8x8mm package, 0.5mm pitch; https://www.xilinx.com/support/documentation/user_guides/ug475_7Series_Pkg_Pinout.pdf#page=260, NSMD pad definition Appendix A"
+  pkg_width: 8
```

Comment:
All are missing the additional_tags item which I thought we were going to put on every one, right? Did I misunderstand?

Reply:
You are right.
I do not know why but I thought the versions with only one name were not included with the additional tags.
Will have this in mind!